components/studio/bin/hab-studio.sh

```diff
@@ -420,7 +420,11 @@ new_studio() {
       $bb mount $v -t proc proc $HAB_STUDIO_ROOT/proc
     fi
     if ! $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/sys type"; then
-      $bb mount $v --rbind /sys $HAB_STUDIO_ROOT/sys
+      if [ -z "${KRANGSCHNAK+x}" ]; then
```

Comment:
What's the `+x` do?

Reply:
Checks for string rather than just is set/not set. `-z` checks for non-zero length.